example/extending/plugins/7-writing-mill-plugins/build.mill

```diff
@@ -14,7 +14,7 @@ object myplugin extends ScalaModule with PublishModule {
   def platformSuffix = "_mill" + mill.util.BuildInfo.millBinPlatform
 
   // Depend on `mill-dist` so we can compile against Mill APIs
-  def mvnDeps = Seq(mvn"com.lihaoyi:mill-dist:$millVersion")
+  def mvnDeps = Seq(mvn"com.lihaoyi::mill-main:$millVersion")
```

Comment:
Can you also update the comment above?

Reply:
Did so in https://github.com/com-lihaoyi/mill/commit/78e7c103e03e26f1b6af7b06a90b2ab1cb8f9376

Reply:
Oops I think I edited the wrong branch, nevermind once this is merged it should be ok